types/conda-definition.json

```diff
@@ -9,36 +9,44 @@
     "default_repository_url": "https://repo.anaconda.com"
   },
   "namespace_definition": {
-    "note": "there is no namspace",
+    "note": "There is no namespace for Conda packages.",
     "requirement": "prohibited"
   },
   "name_definition": {
+    "requirement": "required",
```

Comment:
What would it mean for a package to not have a name? What meaning could that PURL provide?